src/components/Color.tsx

```diff
@@ -15,31 +16,36 @@ const methods = [
 	'bgHsv',
 	'bgHwb',
 	'bgRgb',
-	'bgKeyword'
+	'bgKeyword',
+	'ansi',
+	'ansi256',
+	'bgAnsi',
+	'bgAnsi256'
 ];
 
-const Color: FC<ColorProps & { children: ReactNode }> = ({children, ...colorProps}) => {
+const Color: FC<ColorProps & { children: ReactNode }> = ({
+	children,
+	...colorProps
+}) => {
 	if (children === '') {
 		return null;
 	}
 
-	// TODO: Figure out what's going on here and write the type definition
-	const transformChildren = (children: ReactNode) => {
-		Object.keys(colorProps).forEach(method => {
-			// @ts-ignore
-			if (colorProps[method]) {
-				if (methods.includes(method)) {
-					// @ts-ignore
-					children = chalk[method](...arrify(colorProps[method]))(children);
-					// @ts-ignore
-				} else if (typeof chalk[method] === 'function') {
-					// @ts-ignore
-					children = chalk[method](children);
+	const transformChildren = (children: string) => {
+		return Object.keys(colorProps).reduce(
```

Comment:
The forEach was much more readable. Generally, `reduce` is not the right solution unless you actually want to reduce numbers. Use a `for-of` loop.

Reply:
I don't agree with both of those statements but I'll use `for-of` because it's not my codebase.

Reply:
I'm not able to use `for-of` because of https://github.com/Microsoft/TypeScript/issues/3500, so I restored the original forEach loop.

Reply:
> I don't agree with both of those statements but I'll use `for-of` because it's not my codebase.

https://twitter.com/jaffathecake/status/1213077702300852224?s=21